### DISCUSSION TITLE
DependencyNotFoundError error using latest version

### DISCUSSION THREAD

**Discussion by molinto** [uber/react-map-gl#2164] [General]
Hi, I'm trying to create a Map with polygon/line editing. I've forked an example from 'react-map-gl-draw':
[https://codesandbox.io/s/react-map-gl-draw-example-forked-lzxvfn?file=/package.json](https://codesandbox.io/s/react-map-gl-draw-example-forked-lzxvfn?file=/package.json)

When I try to update the:  "react-map-gl": "^**6.1.21**" to "^**7.0.21**" I get the error:

_**DependencyNotFoundError: Could not find dependency: 'mapbox-gl' relative to '/node_modules/react-map-gl/dist/esm/components/map.js'**_

Any help great appreciated :)

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/docs/upgrade-guide#upgrading-to-v70

**Reply by molinto** [reply]
Thanks for the reply, I've been through this. I've added '"mapbox-gl": "^2.10.0"' to the package json on the codesandbox.io fork.
Still have errors
